README.md

```diff
@@ -50,6 +59,7 @@ Don't forget to also add S3 IAM permissions for Spark to be able to fetch table
   - Hadoop 2.8.5
   - Hive 1.2.1
   - AWS SDK 1.11.682
+  - Bigquery Connector 0.18.1
```

Comment:
`0.19.0` is out since yesterday providing the connector with a [Spark's Datasource V2 API implementation](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/pull/283/files) for its backend. Huge improvement in perspective. See [this video](https://databricks.com/session/apache-spark-data-source-v2) if you want to have details about DataSource v1 vs Data Source v2